### PR TITLE
[6.2.0]Expose cc_proto_aspect as a toplevel symbol.

### PR DESCRIPTION
**Pull request by ShreeM01**
Commit: [2aa06cf](https://github.com/bazelbuild/bazel/commit/2aa06cf227fe349195191107286167035f0c5431)

PiperOrigin-RevId: 520619683
Change-Id: Ief39fdc3ea9083c22fbf12d2b4ba79482c3c16bf

### CODE REVIEW

**Comment by keertk** [comment]
@meteorcloudy It seems like part of another commit (https://github.com/bazelbuild/bazel/commit/1b1922a9cd5d33f7c50461235b55cffa85f4ac90#diff-91bc31bbfbe0bcc74ea670f356a695600f5771b33875f6629beee3a7cef20f96) likely needs to be cherry-picked first? 

I removed part of the change from `src/main/starlark/builtins_bzl/bazel/exports.bzl` due to failures but can re-add once you confirm.

**Review by buildbreaker2021 — APPROVED** [review]

**Comment by meteorcloudy** [comment]
Wait, @buildbreaker2021, there are some changes missing, I guess we need to cherry pick https://github.com/bazelbuild/bazel/commit/1b1922a9cd5d33f7c50461235b55cffa85f4ac90 as well?

File: src/main/starlark/builtins_bzl/bazel/exports.bzl [hunk 1]
```diff
@@ -17,7 +17,7 @@
 load("@_builtins//:common/java/java_library.bzl", "java_library")
 load("@_builtins//:common/java/java_plugin.bzl", "java_plugin")
 load("@_builtins//:common/java/proto/java_proto_library.bzl", "java_proto_library")
-load("@_builtins//:common/cc/cc_proto_library.bzl", "cc_proto_library")
+load("@_builtins//:common/cc/cc_proto_library.bzl", "cc_proto_aspect", "cc_proto_library")
 
 exported_toplevels = {}
```
Comment (meteorcloudy) [inline src/main/starlark/builtins_bzl/bazel/exports.bzl:22]: Here is missing ` "cc_proto_aspect": cc_proto_aspect,` @buildbreaker2021

Comment (meteorcloudy) [reply src/main/starlark/builtins_bzl/bazel/exports.bzl:22]: Maybe just adding this back without https://github.com/bazelbuild/bazel/commit/1b1922a9cd5d33f7c50461235b55cffa85f4ac90 is enough?

Comment (buildbreaker2021) [reply src/main/starlark/builtins_bzl/bazel/exports.bzl:22]: Please address this.

**Comment by keertk** [comment]
@buildbreaker2021 just saw your comments and made the update. Could you confirm and approve?

**Review by buildbreaker2021 — APPROVED** [review]

**Comment by gobeil** [comment]
Isn't it also important to export the key `proto_common.ProtoLangToolchainInfo`, so that once an aspect declares its dependency on `cc_proto_aspect`, it can extract the `CcInfo` by doing:
```
proto_toolchain = ctx.attr._proto_lang_toolchain[proto_common.ProtoLangToolchainInfo]
cc_info = proto_toolchain.runtime[CcInfo]
```
Perhaps I'm missing another way to do that?

**Comment by meteorcloudy** [comment]
> Isn't it also important to export the key proto_common.ProtoLangToolchainInfo,

@buildbreaker2021 @comius WDYT?

**Pull request merged**